cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/DashboardPage.java

```diff
@@ -0,0 +1,269 @@
+package com.skillstorm.PageObjects;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import com.skillstorm.PageObjects.Components.Navbar.DashboardNavbar;
+import com.skillstorm.PageObjects.Components.Navbar.LandingNavbar;
+import com.skillstorm.PageObjects.Components.Navbar.Navbar;
+import com.skillstorm.PageObjects.Interfaces.Component;
+
+public class DashboardPage extends Page {
+//#region Static fields
+
+    //Names
+    public static final String CMP_LANDING_NAVBAR_NAME = "Landing Navbar";
```

Comment:
Low impact: Different naming conventions in same file.

Reply:
fixed